oauth2-server/src/main/java/com/clouway/oauth2/IssueNewTokenActivity.java

```diff
@@ -64,27 +70,26 @@ public Response execute(Client client, Request request, DateTime instant) {
     String certKey = randomKey(certificates);
     Pem.Block key = certificates.get(certKey);
 
-
     TokenResponse response = tokens.issueToken(GrantType.AUTHORIZATION_CODE, client, identity, authorization.scopes, instant);
     if (!response.isSuccessful()) {
       return OAuthError.invalidRequest("Token cannot be issued.");
     }
 
     BearerToken accessToken = response.accessToken;
-    String host = request.header("Host");
 
-    String idToken = newIdToken().header(new Header("RS256"))
-            .claims(newClaimSet()
-                    .iss(host)
-                    .aud(client.id)
-                    .sub(identity.id())
-                    .iat(instant.timestamp())
-                    .exp(accessToken.expirationTimestamp())
-                    .certId(certKey)
-                    .customClaims(identity.claims())
-                    .build())
-            .signWith(key)
-            .build();
+    String host = request.header("Host");
+    DefaultJwtBuilder idTokenBuilder = new DefaultJwtBuilder();
+
+    String idToken = idTokenBuilder
```

Comment:
You cab shorten it with the static builder method: Jwts.builder().....setHeaderParam();

This will remove the instantiation.